lib/locales/it.js

```diff
@@ -45,20 +45,20 @@ module.exports = {
         'Taking snapshot - this may take a few seconds': 'Creazione screenshot - potrebbe richiedere qualche secondo', // components/modebar/buttons.js:57
```

Comment:
The community member suggests "Facendo screenshot - ...." instead of "Creazione screenshot" - what do you think?